setup.cfg

```diff
@@ -66,7 +66,7 @@ enable =
     use-implicit-booleaness-not-comparison-to-string,
     use-implicit-booleaness-not-comparison-to-zero
 ignore = tinytag.py
-ignore-paths = build, venv
+ignore-paths = build, pynicotine/external/data, venv
```

Comment:
It would be interesting to see what errors the linter says.

Reply:
The linter uses too much memory when processing the file.